distro/config.go

```diff
@@ -84,19 +183,45 @@ func (fn optionFunc) apply(c *config) {
 	fn(c)
 }
 
-// WithEndpoint configures the endpoint telemetry is sent to.
-// Setting empty string results in no operation.
+// WithEndpoint configures the endpoint telemetry is sent to. Passing an empty
+// string results in the default value being used.
 func WithEndpoint(endpoint string) Option {
 	return optionFunc(func(c *config) {
 		c.Endpoint = endpoint
 	})
 }
 
-// WithAccessToken configures the authentication token
-// allowing exporters to send data directly to a Splunk back-end.
-// Setting empty string results in no operation.
+// WithAccessToken configures the authentication token used to authenticate
+// telemetry delivery requests to a Splunk back-end. Passing an empty string
```

Comment:
I find the term `directly to Splunk Observability Cloud` more accurate and it is used in the "official docs" e.g. https://docs.splunk.com/Observability/gdi/get-data-in/application/java/instrumentation/instrument-java-application.html#send-data-directly-to-observability-cloud